template.tpl

```diff
@@ -2023,6 +2085,23 @@ switch (data.eventType) {
       return fail('Unable to track any valid Enhanced Ecommerce event.');
     }
     break;
+    
+  case 'enableButtonClickTracking':
+    commandName = data.eventType;
+    
+    // `data.buttonClickTrackingFilterChoice` will always be set,
+    // being "allowlist" by default
+    const choice = data.buttonClickTrackingFilterChoice;
+    const filters = data.buttonClickTrackingFilters
+      .split(",")
+      .map(f => f.trim());
```

Comment:
Doesn't this mean we will always use a `{filter: {allowlist: [""]}}` default and never track anything unless an actual comprehensive list of classes is provided? I don't think [`classes.some((className) => filter.allowlist.includes(className))`](https://github.com/snowplow/snowplow-javascript-tracker/blob/e835323adfe8ec7b8ea084a33fe3a7f34efa1df8/plugins/browser-plugin-button-click-tracking/src/util.ts#L21) will ever succeed with that input?

Should we do this bit later and guard it in a check on `data.buttonClickTrackingFilters.length` or something?

Reply:
Yeah, I've fiddled with this a bit now. The command/parameters are only set if the value for filter is `undefined` (no filter set), or we can parse a sensible array of strings out of the value provided.